File: 프로그래머스/0/181942. 문자열 섞기/문자열 섞기.java
class Solution {
    public String solution(String str1, String str2) {
//         String answer = "";
        
//         for(int i = 0; i < 2 * str1.length(); i++){
//             if(i % 2 == 0){
//                 answer += str1.charAt(i / 2);
//             }else{
//                 answer += str2.charAt(i / 2);
//             }
//         }
        
//         return answer;
        
        StringBuilder sb = new StringBuilder();
        
        for(int i = 0; i < 2 * str1.length(); i++){
            if(i % 2 == 0){
                sb.append(str1.charAt(i / 2));
            }else{
                sb.append(str2.charAt(i / 2));
            }
        }
        
        return sb.toString();
    }
}

File: 프로그래머스/0/181942. 문자열 섞기/문자열 섞기.py
def solution(str1, str2):
    answer = []
    
    for i in range(len(str1) * 2):
        if i % 2 == 0:
            answer.append(str1[i // 2])
        else:
            answer.append(str2[i // 2])
    
    return "".join(answer)